src/staticanalysis/bot/static_analysis_bot/clang/format.py

```diff
@@ -52,7 +52,7 @@ def get_eligible_files():
             for file in revision.files:
                 # Verify if file is clang-format compliant, meaning that's a C/C++
                 _, ext = os.path.splitext(file)
-                if ext.lower() in settings.cpp_extensions:
+                if ext.lower() in settings.cpp_extensions or ext.lower() in settings.cpp_header_extenssions:
```

Comment:
You can do `ext.lower() in settings.cpp_extensions + settings.cpp_header_extensions`

Reply:
sure but i thought it a little bit more visible.